src/gradualizer_cli.erl

```diff
@@ -65,27 +65,27 @@ print_usage() ->
     io:format("  -h,  --help                    display this help and exit~n"),
     io:format("       --infer                   Infer type information from literals and other~n"),
     io:format("                                 language constructs~n"),
-    io:format("       --no-infer                Only use type information from function specs~n"),
+    io:format("       --no_infer                Only use type information from function specs~n"),
     io:format("                                  - the default behaviour~n"),
     io:format("       --verbose                 Show what Gradualizer is doing~n"),
-    io:format("  -pa, --path-add                Add the specified directory to the beginning of~n"),
+    io:format("  -pa, --path_add                Add the specified directory to the beginning of~n"),
```

Comment:
For consistency with `erl` and `erlc`, while keeping the interface small, I think we should remove this alias and instead add `-pz`. Only `-pa` and `-pz` exist for erl and erlc, with no longer alias. (This is a CLI only option. From within Erlang, use `code:add_pathsa/1` and friends.)